sdk/containerinstance/arm-containerinstance/package.json

```diff
@@ -20,18 +20,19 @@
   "module": "./esm/containerInstanceManagementClient.js",
   "types": "./esm/containerInstanceManagementClient.d.ts",
   "devDependencies": {
-    "typescript": "^3.1.1",
-    "rollup": "^0.66.2",
-    "rollup-plugin-node-resolve": "^3.4.0",
-    "uglify-js": "^3.4.9"
+    "typescript": "^3.5.3",
+    "rollup": "^1.18.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-sourcemaps": "^0.4.2",
+    "uglify-js": "^3.6.0"
   },
-  "homepage": "https://github.com/azure/azure-sdk-for-js/tree/master/sdk/containerinstance/arm-containerinstance",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js",
```

Comment:
This looks not quite right. Can you try to re-generate it in your local to see if it changes?